lib/torrent.js

```diff
@@ -35,6 +35,7 @@ import utp from './utp.cjs' // browser exclude
 import WebConn from './webconn.js'
 
 import info from '../package.json' assert { type: 'json' }
+import { Selection } from './selection.js'
```

Comment:
this should be above the info import, right under webconn

Reply:
got it, thanks. fixed it